### ISSUE TITLE
Two clicks are needed to deselect a value and after deselecting a value arrows key navigation in the table does not work

### ISSUE THREAD

**Issue by mlb76** [qlik-oss/sn-table#321]
Precondition:  Use NVDA screenreader and Chrome

1.  Go to a visualisation and choose to view data
2.  Enter a table a column and press space to select the value
3.  Press esc to deselect value -> nothing happens
4.  Press esc again -> value is deselected
5.  Try to move to another row or column using arrow keys

Actual result: Nothing happens. It is not possible to move between columns and rows using arrow keys
